python/PythonMessage.h

```diff
@@ -0,0 +1,110 @@
+/*
+ * This file is part of libArcus
+ *
+ * Copyright (C) 2016 Ultimaker b.v. <a.hiemstra@ultimaker.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef ARCUS_MESSAGE_PTR_H
+#define ARCUS_MESSAGE_PTR_H
+
+#include <Python.h>
+#include "Types.h"
+
+namespace google
```

Comment:
Might want to mention that this is because of forward declaration (as there could be other reasons why you are doing this)